cmd/agent-js-worker/main.go

```diff
@@ -496,9 +505,21 @@ func agentOpts(opts *agentStartOpts) ([]aries.Option, error) {
 		options = append(options, aries.WithTransportReturnRoute(opts.TransportReturnRoute))
 	}
 
-	store, err := jsindexeddb.NewProvider(opts.DBNamespace)
-	if err != nil {
-		return nil, err
+	var store storage.Provider
+
+	if opts.SDSServerURL == "" {
```

Comment:
please use a "storage type" field to switch on.

Reply:
Done